spec.md

```diff
@@ -2562,6 +2562,11 @@ message NodeExpandVolumeRequest {
   // access_type from given volume_path for the volume and perform
   // node expansion. This is an OPTIONAL field.
   VolumeCapability volume_capability = 5;
+
+  // Secrets required by plugin to complete node expand volume request.
+  // This field is OPTIONAL. Refer to the `Secrets Requirements`
+  // section on how to use this field.
+  map<string, string> secrets = 6 [(csi_secret) = true];
```

Comment:
We want to make sure all new fields, methods, etc. go through alpha/beta/ga.

So please add `(alpha_field) = true` to this field to indicate that it is alpha in this release, in the next release we can move to beta, and GA after that.

Reply:
```suggestion
  map<string, string> secrets = 6 [(csi_secret) = true, (alpha_field) = true];
```